applet/src/addl_config.h

```diff
@@ -21,10 +21,10 @@ typedef struct addl_config {
     uint8_t micbargraph;
     uint8_t netmon;
     uint8_t rbeep;
-    uint8_t bootscr;
+    uint8_t boot_demo;
+    uint8_t boot_splash;
     uint32_t dmrid;
```

Comment:
I may have inadvertently changed reordered it when I add bootscr... I'll move it back after dmrid if that is ok with you.